onebusaway-api-webapp/src/main/java/org/onebusaway/api/actions/api/where/ArrivalsAndDeparturesForLocationAction.java

```diff
@@ -58,6 +60,8 @@ public void setFilterChain(FilterChain filterChain) {
         _query.setSystemFilterChain(filterChain);
     }
 
+    @Autowired
+    private RouteSort subwayRouteSort;
```

Comment:
lets not call this subwayRouteSort.  We have a mental model issue here.

If RouteSort is always going to be present, then it should be defaultSort.  If RouteSort is only optionally present, then we need to test for null and not sort if empty